src/main/resources/assets/admin/common/js/form/set/FormSetOccurrenceView.ts

```diff
@@ -225,13 +229,19 @@ export abstract class FormSetOccurrenceView
     }
 
     showContainer(show: boolean) {
```

Comment:
Rename this to `setContainerVisible`, since it can also hide, not just show. Also rename "`show`" argument to "`visible`".